.storybook/main.js

```diff
@@ -30,7 +30,7 @@ module.exports = {
     return {
       ...config,
       // Creates the icon names environment variable for the component-library/icon/icon.js component
-      ICON_NAMES: await generateIconNames(),
+      ICON_NAMES: generateIconNames(),
```

Comment:
`generateIconNames` is no longer an async function so updating here. Icons still work

### Before
<img width="1440" alt="Screenshot 2022-12-11 at 7 50 07 PM" src="https://user-images.githubusercontent.com/8112138/206958473-132d00b6-db65-4665-90c1-4c2763226459.png">


### After
<img width="1440" alt="Screenshot 2022-12-11 at 7 49 22 PM" src="https://user-images.githubusercontent.com/8112138/206958468-21999c9f-47de-4ae5-962e-90f2e4d4400b.png">